### ISSUE TITLE
browser URLs for searching authorities don't work

### ISSUE THREAD

**Issue by graybeal** [mmisw/mmiorr#346]
Searching for an authority works, but the generated URL (https://mmisw.org/orr/#ba/agu) fails when used as the browser URL. The authority search based on ont (#321) also fails.

The ST (search terms) URL is working as expected.

Reported by Emilio M:

```
BTW, I noticed only after sending my message that url's such as http://mmisw.org/orr/#ba/ioos 
actually don't seem to work. They display on the browser address bar when one selects an 
authority, but if you type the url to go to it, you only get an MMI ORR banner, with no real
 content.
```

Reference #321.

**Comment by graybeal** [comment]
Note: If the ont URL alone is hard to fix, option B (acceptable to Emilio) is to redirect that pattern to the orr search. But I'm guessing they both fail for the same reason, an ontology I added for a cuahsi test (likely https://mmisw.org/orr/#http://www.isi.edu/ikcap/geosoft/ontology/csdms).

**Comment by carueda** [comment]
duplicate of #321
